patterns/contact-centered-social-link.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Title: Centered link and social links
+ * Slug: twentytwentyfive/contact-centered-social-link
+ * Keywords: contact, social-link
+ * Categories: contact
+ *
+ * @package Twenty Twenty-Five
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80)">
+	<!-- wp:paragraph {"align":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}}},"textColor":"contrast","fontSize":"x-large"} -->
```

Comment:
Contrast is the default text color, so it can be removed from the pattern. Link color has not been added to the theme yet, but can also be removed.

Reply:
Thanks @carolinan I removed the text and link color.